docs/features.md

```diff
@@ -1,65 +1,167 @@
-2. The JSON report is parsed and the resolved dependencies are normalized and mapped to the configured conda channels via different sources (e.g. `cf-graph-countyfair`, `grayskull`, `parselmouth`).
-3. The packages that were found on the configured conda channels are installed with `conda`. Those _not_ on conda are installed individually with `pip install --no-deps`.
+The install command takes PyPI packages and converts them to `.conda` format
+when possible, falling back to direct pip installation when needed. The
+process begins with dependency resolution using pip's own dependency resolver
+to determine what packages need to be installed.
 
-:::{admonition} Coming soon
-:class: seealso
+Explicitly requested packages are always installed from PyPI and converted
+to `.conda` format to ensure you get exactly what you asked for. For
+dependencies, conda-pypi intelligently chooses the best source using a
+conda-first approach. If a dependency is available on conda channels, it will
+be installed with `conda` directly. If not available on conda channels, the
+dependency will be converted from PyPI to `.conda` format.
 
-Right now we are not disallowing compiled wheels, but we might add options in the future to only allow pure Python wheels via `whl2conda`.
-:::
+The system uses multiple sources for package name mapping, including
+automated mappings from the grayskull project, name normalization that
+handles dash and underscore variants, and direct searches of configured conda
+channels for equivalent packages. VCS and editable packages are handled as
+special cases and installed directly with `pip --no-deps`.
+
+You can preview what would be installed without making changes using
+`--dry-run`, install packages in editable development mode with `--editable`
+or `-e`, and force dependency resolution from PyPI without using conda
+channels using `--override-channels`.
+
+### `conda pypi convert`
+
+The convert command transforms PyPI packages to `.conda` format without
+installing them, which is useful for creating conda packages from PyPI
+distributions or preparing packages for offline installation. You can specify
+where to save the converted packages using `-d`, `--dest`, or `--output-dir`.
+The command supports converting multiple packages at once and can skip conda
+channel checks entirely with `--override-channels` to convert directly from
+PyPI.
+
+Here are some common usage patterns:
+
+```bash
+# Convert packages to current directory
+conda pypi convert requests packaging
+
+# Convert to specific directory
+conda pypi convert -d ./my_packages requests packaging
+
+# Convert without checking conda channels first
+conda pypi convert --override-channels some-pypi-only-package
+```
+
+## PyPI-to-Conda Conversion Engine
+
+`conda-pypi` includes a powerful conversion engine that enables direct
+conversion of Python wheels to `.conda` packages with proper translation of
```

Comment:
All wheels? Or just pure Python wheels?

Reply:
Good catch, I updated this to pure Python wheels. I'm not sure that a check exists, but that should be a feature. I'll open a new issue.